src/scss/skin-modern/components/_subtitleoverlay-cea608.scss

```diff
@@ -0,0 +1,29 @@
+// sass-lint:disable force-element-nesting
+.#{$prefix}-ui-subtitle-overlay.#{$prefix}-cea608 {
+  align-items: flex-end;
+  bottom: unset;
+  display: flex;
+  height: 86%;
+  justify-content: center;
+  left: unset;
+  margin: auto 10%;
+  pointer-events: none;
+  right: unset;
+  top: 50%;
+  transform: translate(0%, -50%);
+  transition: height $animation-duration-short ease-out;
+  width: 80%;
```

Comment:
Increasing the width and at the same time increasing the letter-spacing does not gain us any space. It just makes the text look odd. 

Please remove the letter-spacing. I think you then end up with something similar to [this guide](https://dvcs.w3.org/hg/text-tracks/raw-file/default/608toVTT/608toVTT.html#positioning-in-cea-608)?